riot/lol/constants.go

```diff
@@ -38,14 +38,14 @@ const (
 	endpointSpectatorBase                      = endpointBase + "/spectator/v5"
 	endpointGetCurrentGame                     = endpointSpectatorBase + "/active-games/by-summoner/%s"
 	endpointGetFeaturedGames                   = endpointSpectatorBase + "/featured-games"
-	endpointTournamentStubBase                 = endpointBase + "/tournament-stub/v4"
+	endpointTournamentStubBase                 = endpointBase + "/tournament-stub/v5"
 	endpointCreateStubTournamentCodes          = endpointTournamentStubBase + "/codes?count=%d&tournamentId=%d"
 	endpointGetStubLobbyEvents                 = endpointTournamentStubBase + "/lobby-events/by-code/%s"
 	endpointCreateStubTournamentProvider       = endpointTournamentStubBase + "/providers"
 	endpointCreateStubTournament               = endpointTournamentStubBase + "/tournaments"
-	endpointTournamentBase                     = endpointBase + "/tournament/v4"
+	endpointTournamentBase                     = endpointBase + "/tournament/v5"
 	endpointCreateTournamentCodes              = endpointTournamentBase + "/codes?count=%d&tournamentId=%d"
-	endpointGetLobbyEvents                     = endpointTournamentBase + "/lobby-events/by-code/%s"
+	endpointGetLobbyEvents                     = endpointTournamentBase + "/lobby-events/by-code/%s" 
```

Comment:
nitpick, but could you please remove this change.

Reply:
Sorry, fat fingers. Fixed it, but I will squash the commits as well. So you can wait to trigger a new build 😃